spacelift/data_azure_integrations.go

```diff
@@ -0,0 +1,128 @@
+package spacelift
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+)
+
+func dataAzureIntegrations() *schema.Resource {
+	return &schema.Resource{
+		Description: "`spacelift_azure_integrations` represents a list of all the Azure integrations in " +
```

Comment:
I believe this should be similar to the AWS integrations description, so instead of all the integrations it should be "all the integrations in the Spacelift account visible to the API user".